config.js

```diff
@@ -496,6 +507,12 @@ function getArgNameStringsGRU(familyNameString) {
     return ["", "ranked", "unranked"].map( e => `${familyNameString}${e}` );
 }
 
+function getParsedDaysNumHoursMinutesObj(arg) {
+    const [daysNum, hh, mm] = arg.split(":")
+                                 .map( e => Number(e) );
+    return { daysNum, hh: hh % 24, mm: mm % 60 };
```

Comment:
i would min(hh,23) min(mm,59) instead for edge cases

Reply:
refactored it, can you look at the new code again ?